downstream/modules/platform/ref-controller-user-teams.adoc

```diff
@@ -2,16 +2,16 @@
 
 = Users - Teams
 
-From the *Users > Details* page, select *Organizations* tab to display the list of teams of which that user is a member. 
+From the *Users > Details* page, select the *Teams* tab to display the list of teams of which that user is a member. 
 
 You can search this list by *Team Name* or *Description*. 
 
 [NOTE]
 ====
-Team membership cannot be modified from this display panel. 
-For more information, see [Teams].
+You cannot modify team membership from this display panel. 
+For more information, see xref:assembly-controller-teams[Teams].
 ====
 
-Until a Team has been created and a user has been assigned to that team, the assigned Teams Details for that User appears empty.
+Until you create a team and assign a user to that team, the assigned teams details for that user appears empty.
```

Comment:
```suggestion
Until you create a team and assign a user to that team, the assigned teams details for that user is displayed as empty.
```
IBM Style Guide discourages the use of "appear": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a . (See "appear" in the list which states, "Do not use to refer to dialog boxes, windows, or wizards. Use open.")

Example of using display: "Use only as a transitive verb. For example, write “the system displays a message” or “the message is displayed” (not “the message displays”).